modules/publication/test/TestPlan.md

```diff
@@ -0,0 +1,40 @@
+# Publication - Test Plan
+1. If not already sourced into your DB sandbox, source the publication 
+schema into your DB sandbox and make sure no errors arise.
+2. Before granting your (non-super / admin) user account any of the 
```

Comment:
Your patch gives the admin these permissions. Are you going to remove that after dev?

Reply:
no, i think permissions can only be granted to users by a user who themselves have that permission
i think it should be kept in the initial patch so projects don't have to make a separate patch to make the permissions accessible